### DISCUSSION TITLE
六月更新 - 定

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1243] [通知]
项目重构完成，已基本稳定，可以日常使用。下个月的目标是做 XBOX 版本

## 🚀 功能

- 添加回退快捷操作 @Richasy (#1233)

## 🐛 Bug 修复

- 修复登录按钮失效的问题 @Richasy (#1242)
- 修复应用启动时未加载用户资料的问题 @Richasy (#1241)
- 修复主题切换不生效的问题 @Richasy (#1239)
- 修复侧键返回的触发 @Richasy (#1235)
- 修复音量未正确初始化的问题 @Richasy (#1234)
- 修复无法登录的问题 @Richasy (#1228)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2206.3.0'>六月更新 - 定</a>.</em>

**Comment by VictorNanka** [comment]
这个最终版本会发布到应用商店吗？

**Reply by Richasy** [reply]
不会，等 XBOX 做完了一起发

**Comment by ghost** [comment]
我还以为大佬以后都不更新了，辛苦了

**Reply by ghost** [reply]
好像有个bug如果把鼠标指针放在进度条上不动，进度条也会停在那不动，但是画面是正常播放的

**Reply by Richasy** [reply]
@ConfusedOneself 这是预期的行为，目的是为了让人在调整进度的时候有一个明确的起始位置

**Reply by ghost** [reply]
懂了，感谢作者大大的解答

**Reply by ghost** [reply]
还有bug就是，当哔哩在后台运行播放视频，过一会再切换回前台，弹幕就会有卡顿

**Comment by FengLiuI** [comment]
大佬，我说一下目前这个版本发现的两个小问题：
1.全屏播放视频时，软件是圆角的，但屏幕是直角的，所以屏幕的四个角就有空白
2.我开了全局播放速率，但是看直播好像也是二倍速，所以就一卡一卡的。视频那边改成一倍速，看直播依然是二倍速

**Reply by Richasy** [reply]
问题请发issue

**Comment by CsyBarry** [comment]
以后会有Linux系统的版本吗

**Reply by CsyBarry** [reply]
甚至可以直接apt-get install

**Reply by Richasy** [reply]
不可能了，这是UWP，只在Windows平台

**Reply by driver1998** [reply]
理论上可以走Uno Platform跨平台，但那东西并不算特别高...
